Hyper/Meshes/Mesh.cs

```diff
@@ -10,6 +10,9 @@ internal class Mesh : IDisposable
 {
     public Vector3 Position { get; set; }
 
+    public Vertex[] vertices;
+    //Will also have to add rotation and scale
```

Comment:
is this still relevant? We have `Scaling` and `RigidPose` which has rotation

Reply:
i do not think the comment is mine, vertices field is in fact completely unnecessary